growth/too/tasks/scheduler.py

```diff
@@ -14,8 +14,10 @@
 from . import celery
 from .. import models
 from .. import views
+from flask import flash
 
-ZTF_URL = 'http://tunnel:9999'
+
+ZTF_URL = 'http://localhost:9999'
```

Comment:
This looks like a change for local testing that was included by accident.

Reply:
ARGHHHHHHHHHHHHH. Fixed. Thanks Leo.